tests/zfs-tests/cmd/xattrtest/xattrtest.c

```diff
@@ -168,6 +168,7 @@ parse_args(int argc, char **argv)
 			break;
 		case 'p':
 			strncpy(path, optarg, PATH_MAX);
+			path[PATH_MAX - 1] = '\0';
```

Comment:
You'll want to update the `-t` case a few lines down which suffers from the exact same issue.